src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -1190,6 +1190,18 @@ declare global {
 
 Astro reserves the `astro:` prefix for future built-in hooks. Please choose a different prefix when naming your custom hook.
 
+## Astro vite environments
+
+Since v6, Astro is based on `vite` v7 and it heavily uses its [environment APIs](https://vite.dev/guide/api-environment). 
+  
+Astro build environments:
+- `ssr`: the classic and default `ssr` environment of `vite`. Astro uses this environment to build dynamic routes.
```

Comment:
It's also used for non-dynamic routes in dev.

Reply:
If the goal is to explain that Astro uses environments now I would probably explain it all together. If the goal is to explain how there's `prerender` as a special environment in the build, I would explain the reasoning for this documentation more. Like for example something like:

```md
Astro inherits the environments Vite provides by default, `ssr` and `client`. Additionally there is the the `prerender` environment provided by Astro. Here's what each of these environments are used for:
```